docs/dev/devel-env.rst

```diff
@@ -286,12 +294,152 @@ Also, run the **integration tests** for ListenBrainz.
 When the tests complete, you will see if your changes are valid or not. These tests
 are a helpful way to validate new changes without a lot of work.
 
-FAQ's
+Set up Listenbrainz Spark development environment
+=================================================
+
+The Listenbrainz Spark module is used to generate recommendations and stats using Apache Spark. The recommendations are generated based on collaborative filtering technique.
+
+To contribute to the ListenBrainz Spark project, you need a development environment.
+With your development environment, you can test your changes before submitting a
+patch back to the project. This guide helps you set up a development environment
+and run ListenBrainz Spark locally on your workstation. By the end of this guide, you
+will have…
+
+* Install system dependencies
```

Comment:
Installed?

Reply:
Are they automatically installed? I guess this section mentions the steps needed to install required dependencies.